dsl/src/nativeMain/kotlin/com/freeletics/flowredux/dsl/util/AtomicInt.kt

```diff
@@ -0,0 +1,21 @@
+package com.freeletics.flowredux.dsl.util
+
+import kotlin.native.concurrent.AtomicInt
+
+public actual class AtomicInt actual constructor(initialValue: Int) {
+    private val atom = AtomicInt(initialValue)
```

Comment:
should this use full package name to avoid resolving to the "wrong" ˋAtomicIntˋ?

Reply:
> 

Comment above could be obsolete if we rename class to AtomiCounter as proposed in another comment above

Reply:
👍🏻Renamed